modules/cudf/src/node_cudf/column.hpp

```diff
@@ -722,6 +722,28 @@ struct Column : public EnvLocalObjectWrap<Column> {
     std::string const& pattern,
     rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource()) const;
 
+  // column/convert.cpp
+
```

Comment:
Extreme nit pick, typically we'd remove this new line.

Reply:
not sure where that came from!